lib/poseidon/sync_producer.rb

```diff
@@ -57,6 +57,7 @@ def send_messages(messages)
         if !messages_to_send.pending_messages? || @max_send_retries == 0
           break
         else
+          Poseidon.logger.debug { "retrying sending #{messages_to_send.messages} messages"  }
```

Comment:
I'm curious what `messages_to_send.messages` will look like when `to_s`d. I think there's a distinct change it's a bunch of `BSON.serialized` junk.